src/utils/tokenHelpers.ts

```diff
@@ -16,3 +17,23 @@ export const getAllTokenIdsForChain = memoize(
       `${String(args[0])}|${config.tokens.lastUpdate.getTime()}`,
   },
 );
+
+const normalizeAddress = (value: string): string => {
+  return value.startsWith('0x') ? value.toLowerCase() : value;
+};
+
+export const isCanonicalUSDCToken = (
```

Comment:
Maybe these (and any specific usdc stuff) can be moved to utilt/usdc.ts so that the next person knows where to get usdc stuff